vfcompositor/Compositor.c

```
@@ -0,0 +1,784 @@
+    
+    // Free resources
+    if (window->back_buffer) {
+        KernelFree(window->back_buffer);
+    }
+    if (window->title) {
+        KernelFree((void*)window->title);
+    }
+    KernelFree(window);
+}
+
+void WindowFill(Window* window, uint32_t color) {
```

Clang (14.0.6)

[warning] parameter 'window' is unused

(misc-unused-parameters)